README.md

```diff
@@ -7,6 +7,10 @@ A version of the [PORTA](http://porta.zib.de/) software intended for cross-platf
 
 "PORTA is a collection of [C] routines for analyzing polytopes and polyhedra. The polyhedra are either given as the convex hull of a set of points plus (possibly) the convex cone of a set of vectors, or as a system of linear equations and inequalities." -([PORTA](http://porta.zib.de/)).
 
+## Quick Start
+
+TODO: Add compilation instructions 
```

Comment:
Please add instructions for how to compile porta from source using CMAKE

Reply:
Ahh I meant to ask where we wanted to put that. Once the installation is set in stone (maybe more hoops to jump through if the wizard doesn't work the first time thru) I'll add that here and remove the current instructions from `INFO`

Reply:
odne